sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/schemas/Sgv2IndexesResourceImpl.java

```diff
@@ -0,0 +1,177 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.sgv2.restsvc.resources.schemas;
+
+import io.stargate.grpc.StargateBearerToken;
+import io.stargate.proto.QueryOuterClass.Query;
+import io.stargate.proto.Schema;
+import io.stargate.proto.StargateGrpc;
+import io.stargate.sgv2.common.cql.builder.Predicate;
+import io.stargate.sgv2.common.cql.builder.QueryBuilder;
+import io.stargate.sgv2.restsvc.grpc.BridgeSchemaClient;
+import io.stargate.sgv2.restsvc.impl.GrpcClientFactory;
+import io.stargate.sgv2.restsvc.models.Sgv2IndexAddRequest;
+import io.stargate.sgv2.restsvc.resources.ResourceBase;
+import io.stargate.sgv2.restsvc.resources.Sgv2RequestHandler;
+import java.util.Collections;
+import java.util.Map;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("/v2/schemas/keyspaces/{keyspaceName}/tables/{tableName}/indexes")
+@Produces(MediaType.APPLICATION_JSON)
+@Singleton
+public class Sgv2IndexesResourceImpl extends ResourceBase implements Sgv2IndexesResourceApi {
+
+  @Inject private GrpcClientFactory grpcFactory;
+
+  @Override
+  public Response getAllIndexesForTable(
+      String token, String keyspaceName, String tableName, HttpServletRequest request) {
+    if (isAuthTokenInvalid(token)) {
+      return invalidTokenFailure();
+    }
+    return Sgv2RequestHandler.handleMainOperation(
+        () -> {
+          if (isStringEmpty(keyspaceName)) {
+            return jaxrsBadRequestError("keyspaceName must be provided").build();
+          }
+          if (isStringEmpty(tableName)) {
+            return jaxrsBadRequestError("tableName must be provided").build();
+          }
+
+          StargateGrpc.StargateBlockingStub blockingStub =
+              grpcFactory
+                  .constructBlockingStub()
+                  .withCallCredentials(new StargateBearerToken(token));
+
+          BridgeSchemaClient.create(blockingStub).findTable(keyspaceName, tableName);
+
+          String cql =
+              new QueryBuilder()
+                  .select()
+                  .from("system_schema", "indexes")
+                  .where("keyspace_name", Predicate.EQ, keyspaceName)
+                  .where("table_name", Predicate.EQ, tableName)
+                  .build();
+          return fetchRows(blockingStub, -1, null, true, cql, null);
```

Comment:
Per the v1 implementation.
This is inconsistent with other DDL resources, which return dedicated representations rather than the raw contents of the system table. But if we want full backward compatibility this is it.

Reply:
Should we add a comment to note this is for backward compatibility?

Reply:
Could update this google doc:

https://docs.google.com/document/d/159QfQ1X3BbDErBNB5QSkg3f7j73RW2dbwD8zVZ90sGc/edit?pli=1

or create new one.

I agree we should collect open issues/concerns somewhere.

Reply:
Suggest we move the contents of that gdoc to a discussion post

Reply:
@jeffreyscarpenter +1 -- does copy-paste work well between the two?

Reply:
I recommend the Google Docs plugin called "Docs to Markdown" by Ed Bacher which you can get in the Google workspace marketplace.